index.html

```diff
@@ -1,327 +1,339 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Letaru Linda Masia</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-  </head>
-  <body>
-    <header class="site-header">
-      <a href="#"> Letaru Linda Masia</a>
-      <nav class="nav-menu">
-        <img src="./images/Vector.png" alt="Vector image " />
-      </nav>
-    </header>
-    <main>
-      <section class="container">
-        <div class="text">
-          <h1>Hey There. I'm Linda</h1>
-          <h3>I'm a Software Developer.</h3>
-          <p>
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don’t hestiate to contact me.
-          </p>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Linda Letaru Masia</title>
+</head>
+<body class="mainBody">
+<!-- Navbar -->
+<header class="navbar">
+  <a class="logo" href="#">Linda Letaru</a>
+  <div class="empty"></div>
+  <ul class="navbar-menu">
+    <li><a href="#">Portfolio</a></li>
```

Comment:
As the previous reviewer suggested, kindly adjust the alignments of the project cards to match the figma design. Note that on a 1440px screen which is the size on the Figma design, only 3 cards should appear on a row.

Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

So, let's follow the original vision if it's pretty and works, it will be a wonderful website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187043543-ea15a234-9bcf-4913-aec7-72963ee56159.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/187043587-c129d162-04b3-4834-8193-e65147aa00e4.png)